.github/workflows/promote-content-to-stable.yml

```
@@ -0,0 +1,27 @@
+name: Promote SDK to stable
+
+on:
+  issue_comment:
+    types:
+      - created
+
+permissions:
+  issues: write
+
+jobs:
+  promote-sdk:
+    name: ⬆️ Promote SDK to stable
+    environment: "2204 Branch"
+    runs-on: ubuntu-latest
+    if: |
+      ( !github.event.issue.pull_request )
+      && contains(github.event.comment.body, '/promote ')
+      && contains(github.event.comment.body, ''ffmpeg-2204')
```

This will match both `ffmpeg-2204` and `ffmpeg-2204-sdk` - I wonder if `ffmpeg-2204 ` might work better?

```suggestion
      && contains(github.event.comment.body, ''ffmpeg-2204 ')
```